src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -86,6 +102,59 @@ public void handlesSocket() throws IOException {
     }
 
     /**
+     * BkBasic can handle HTTP persistent connections.
+     * @throws Exception If some problem inside
+     * @checkstyle ExecutableStatementCountCheck (2 lines)
+     */
+    @Test
+    @SuppressWarnings("PMD.DoNotUseThreads")
+    public void handlesPersistentConnection() throws Exception {
+        final int port = 8080;
```

Comment:
@mstzn36 we can't use port 8080, it may just be not available on the build machine and the test will fail. Of course we have to allocate a random port. See how it's done in other tests